### PR TITLE
removed unneeded plugin update

### PR DESCRIPTION
**Pull request by jennuine**
# 🦟 Bug fix

## Summary
When a plugin was added an `Update` was called immediately. With #497 updates are also being called in a loop, which was causing deadlocks since both could be called at the same time. This PR removes the initial update (after the plugin is inserted) since updates are already happening periodically. 

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: include/ignition/gazebo/gui/GuiRunner.hh [hunk 1]
```diff
@@ -48,10 +48,6 @@ class IGNITION_GAZEBO_VISIBLE GuiRunner : public QObject
   /// \brief Destructor
   public: ~GuiRunner() override;
 
-  /// \brief Callback when a plugin has been added.
-  /// \param[in] _objectName Plugin's object name.
-  public slots: void OnPluginAdded(const QString &_objectName);
```
Comment (chapulina) [inline include/ignition/gazebo/gui/GuiRunner.hh:53]: Ahh I forgot that this header is installed, let's leave the function in, but without doing anything.

Comment (chapulina) [reply include/ignition/gazebo/gui/GuiRunner.hh:53]: 9ecc40977235ea31eb9ca1df3f73b7605c685a89

**Review by chapulina — APPROVED** [review]
Thanks for cleaning this up! Works for me. Let's just wait for CI results.

**Review by iche033 — COMMENTED** [review]
ran into this issue on ign-gazebo4 so ported the fix over and confirmed that the issue is fixed.

JointPositionController tests are failing on CI though

**Comment by chapulina** [comment]
> JointPositionController tests are failing on CI though

Should be fixed by waiting for updates in 20cdc9f

**Pull request merged**